### ISSUE TITLE
Personal and public UTXO tracker with BIP 329 (Wallet Labels Export Format) support

### ISSUE THREAD

**Issue by 0xB10C** [0xB10C/project-ideas#1]
I've been thinking about utxo set monitoring for a while. The idea would be to have a tool with a web page and alerting that:

- allows you to watch-only import some addresses, xpubs, utxos, descriptiors, ...
- tracks the balances on these utxos
- allows tagging/naming the UTXOs with BIP-329

motivation:
- monitor your own funds - are they still there?
- monitor e.g. Satoshis coins and receive a notification/email when they move

You should be able to self-host this and have a public instance or a private instance.

Basically a watch-only wallet.
